ext_tables.sql

```diff
@@ -670,6 +670,13 @@ CREATE TABLE tx_styleguide_type (
 	text_1 text
 );
 
+CREATE TABLE tx_styleguide_type_foreign(
```

Comment:
There is some magic in the data generator: 'top level' tables must have structure 'tx_styleguide_somename' (no further _) ... I'll rename the table.